queries.sql

```diff
@@ -252,3 +251,18 @@ WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name
 ORDER BY COUNT(*) DESC
 LIMIT 1;
+
+/* project 5 */
+SELECT COUNT(*) FROM visits where animal_id = 4;
+
+SELECT * FROM visits where vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+/* explain analyze on queries */
+ EXPLAIN ANALYZE SELECT COUNT (*) FROM visits WHERE animal_id=4;
```

Comment:
👏 Great job with the initial analysis and the inclusion of screenshots as the previous reviewer requested. However, there's a specific step to follow as per the project requirements:

- [ ] Before creating the index, the execution time is 904ms, which is less than 1000ms. As stated in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depending%20on,bigger%20than%201000ms.), "Check that by running explain analyze query: - If you get Execution time: X ms and X >= 1000: that should be enough, and you can continue to the project requirements. - If you get Execution time: X ms and X < 1000: please go back to point 3. and repeat until you get a value bigger than 1000ms." 
- [ ] Please remember to run the query again and update your screenshots. 🙏 

To meet this requirement, kindly follow the steps mentioned and update your screenshot accordingly. This will ensure that your analysis aligns perfectly with the project specifications. Keep up the good work! 👍📊🕐